macros/src/types/enum.rs

```diff
@@ -68,6 +68,10 @@ fn format_variant(
     generics: &Generics,
 ) -> syn::Result<()> {
     let variant_attr = VariantAttr::from_attrs(&variant.attrs)?;
+    let variant_attr = VariantAttr {
+        rename_all: variant_attr.rename_all.or(enum_attr.rename_all_fields),
+        ..variant_attr
+    };
```

Comment:
I shadowed the variable to avoid adding a `mut`, do you think this is a good approach?

Reply:
Seems reasonable to me!
However, I think I'd prefer changing the `VariantAttr` constructor instead. Maybe renaming it to `VariantAttr::new`, and having it take `&[Attribute]` as well as `&EnumAttr`. I'm open to alternatives, but I kinda dislike that it takes two steps to create a `VariantAttr` now.

Reply:
Done :D